gap/SnAnBB.gi

```diff
@@ -1,1452 +1,2663 @@
 #############################################################################
 ##
-##  recogSnAnBB.g
+##  AnBB.gi
 ##                              recog package
-##                                                           Maska Law
-##                                                   Alice C. Niemeyer
-##                                                      & 'Akos Seress
+##                                                            Martin Leuner
+##                                                         Sebastian Jambor
+##                                                      & Alice C. Niemeyer
 ##
 ##
-##  Copyright (C) 2006-2008 by the authors.
-##                          The University of Western Australia.
+##  Copyright (C) 2012 by the authors.
+##  Lehrstuhl B f"ur Mathematik, RWTH Aachen and
+##  the University of Western Australia.
+##
 ##  This file is free software, see license information at the end.
 ##
 ##
-##  This file provides code for recognising whether a black box group
-##  with known degree n is isomorphic to A_n or S_n.
+DeclareInfoClass("InfoRecSnAnBB");
+SetInfoLevel( InfoRecSnAnBB, 0 );
+
+
+
+# Function used to check whether a group element is trivial:
+if not IsBound( AnBB_IsOne ) then
+ AnBB_IsOne := IsOne;
+fi;
+
+# Function used to check whether two group elements equal each other:
+if not IsBound( AnBB_Equals ) then
+ AnBB_Equals := EQ;
+fi;
```

Comment:
I mentioned this before orally, but just to make sure it's not forgotten, I'll record it here, too: Instead of `AnBB_Equals`, the code should use `ri!.isequal`; that is, `AnBB_Equals(x,y)` would become `ri!.isequal(x,y)`. Same for `AnBB_IsOne` / `isone`.